### ISSUE TITLE
Accessibility : Post List Improvements 

### ISSUE THREAD

**Issue by jd-alexander** [wordpress-mobile/WordPress-Android#11067]
Below is a list of improvements that can be made to the accessibility of the Post List. 

## Expected behavior

* When you click the `post list author` dropdown menu item it doesn't say what the dropdown does. 
<img src="https://user-images.githubusercontent.com/1509205/72192966-93dce780-33d5-11ea-8783-9e5f8f08ef03.png" width="320">

* The create button should say "create a post" instead of just "create"
<img src="https://user-images.githubusercontent.com/1509205/72193035-c981d080-33d5-11ea-8d63-32b78c9a49e3.png" width="320">

* The more menu item doesn't meet the minimum requirement of `48dp` for a touch target. 
<img src="https://user-images.githubusercontent.com/1509205/72193091-f3d38e00-33d5-11ea-9433-ee39c4360e2e.png" width="320">

Tested on Google Pixel 4 XL, Android 10, WPAndroid `develop` ()
